app/reports/celebration_verbatims.rb

```diff
@@ -0,0 +1,19 @@
+class CelebrationVerbatims < AdminReport
+  def initialize(team, time_periods)
```

Comment:
I noticed having to pass a lot of nil values when calling these reports. This could be avoided by reversing the order and setting team with a default of nil, e.g. `def initialize(time_periods, team = nil)`